pkg/apis/machine/types.go

```diff
@@ -335,6 +353,25 @@ type MachineSetStatus struct {
 
 	// ObservedGeneration
 	ObservedGeneration int64
+
+	// FailedMachines has summary of machines on which lastOperation Failed
```

Comment:
These fields make more sense if they are optional. Can you please align it with upstream changes in this PR.
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/extensions/types.go#L725

Reply:
Okay shall do that.

Reply:
Done.